README.md

```diff
@@ -502,7 +557,11 @@ continue the development of this module.
 
 ## Development
 
+<<<<<<< HEAD
```

Comment:
looks like this was missed during the last rebase

Reply:
Oops. Yep.